### DISCUSSION TITLE
Atualização do VSCode no Linux

### DISCUSSION THREAD

**Discussion by Everton-D-Alves** [community/community#58463] [Packages]
### Select Topic Area

Question

### Body

Atualizei o VSCode no Linux a versão code_1.79.2-1686734195_amd64.deb e está dando problemas no terminal do VSCode.

**Comment by aahouzi** [comment]
Lamentamos saber que você está enfrentando problemas no terminal do VSCode após atualizar para a versão 1.79.2. Aqui estão algumas sugestões para solucionar ou contornar esse problema:

1. Reinicie o VSCode: Em alguns casos, reiniciar o VSCode pode resolver problemas temporários. Tente fechar e abrir novamente o VSCode para ver se o problema persiste.

2. Verifique as extensões instaladas: Algumas extensões podem entrar em conflito com a nova versão do VSCode. Desative temporariamente todas as extensões e verifique se o problema persiste. Se o problema desaparecer, você pode habilitar as extensões uma por uma para identificar qual delas está causando o problema.

3. Verifique as configurações do terminal: Verifique as configurações do terminal no VSCode para garantir que estejam corretas. Verifique se o shell padrão está configurado corretamente e se as configurações de terminal personalizadas estão de acordo com suas preferências.

4. Reverta para uma versão anterior: Se o problema persistir e estiver causando interrupções significativas em seu fluxo de trabalho, você pode considerar a possibilidade de reverter para uma versão anterior do VSCode que estava funcionando corretamente. Você pode desinstalar a versão atual e baixar uma versão anterior do site do VSCode para reinstalá-la.

5. Relate o problema: Se você tiver seguido as etapas acima e o problema persistir, pode ser útil relatar o problema no repositório oficial do VSCode no GitHub ou nos fóruns de suporte da comunidade. Ao relatar o problema, forneça detalhes específicos sobre o problema encontrado, incluindo mensagens de erro, configurações utilizadas e etapas para reproduzir o problema. Isso ajudará a equipe do VSCode a investigar e resolver o problema.

**Comment by Everton-D-Alves** [comment]
Já está funcionando. Deve ser meu computador que estava lento. Obrigado.

Em Ter, 20 de jun de 2023 14:28, Anas Ahouzi ***@***.***>
escreveu:

**Reply by aahouzi** [reply]
Obrigado @Everton-D-Alves, você pode marcar minha resposta como resposta, por favor?

**Comment by otter-computer** [comment]
Hi @Everton-D-Alves, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
